setup.cfg

```diff
@@ -2,4 +2,7 @@
 test = pytest
 
 [tool:pytest]
-addopts = --verbose
\ No newline at end of file
+addopts = --verbose
+
+[flake8]
+ignore = D101,D103
```

Comment:
these ignores are… ignored

Reply:
either way, I'd much prefer to fix this specific docs issue, rather than ignore it